examples/trtllm/monolith/worker.py

```diff
@@ -29,7 +29,7 @@
 from dynamo.llm import KvMetricsPublisher
 from dynamo.runtime import DistributedRuntime, dynamo_endpoint, dynamo_worker
 
-logger.set_level("debug")
+logger.set_level("info")
```

Comment:
Aren't there any more places which are setting debug logging in the examples? For example, router and disagg worker pieces? 
I would run a find and replace for the logger.set_level.

Reply:
Everything are at "info" level, except this occurrence, i.e.
![debug_level](https://github.com/user-attachments/assets/10f6522d-940e-4861-97b2-4a8e7e17ca0e)